packages/leann-core/src/leann/cli.py

```diff
@@ -421,6 +444,7 @@ def load_documents(
         docs_paths: Union[str, list],
         custom_file_types: Union[str, None] = None,
         include_hidden: bool = False,
+        args=None
```

Comment:
lets have type hints here
```suggestion
        args: Optional[Dict[str, Any]] = None
```